### PR TITLE
Upgrade to Spring Boot 2.6.6 and Spring 5.3.18

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -83,8 +83,8 @@
         <failOnMissingWebXml>false</failOnMissingWebXml>
 
         <!-- Dependencies -->
-        <spring.version>5.3.17</spring.version>
-        <spring.boot.version>2.6.5</spring.boot.version>
+        <spring.version>5.3.18</spring.version>
+        <spring.boot.version>2.6.6</spring.boot.version>
```
Comment (tarekoraby) [inline pom.xml:87]:
```suggestion
        <spring.boot.version>2.6.5</spring.boot.version>
```

Let's change Spring version for now until Spring Boot is released.

Comment (knoobie) [reply pom.xml:87]: Boot is going to be released today

Comment (Artur-) [reply pom.xml:87]: There is no point in only upgrading Spring

Comment (tarekoraby) [reply pom.xml:87]: I was thinking just to see if the tests pass right away. But probably they will anyway.

Comment (caalador) [reply pom.xml:87]: Only fusion-endpoint uses the spring version as everything else is defined through the `<artifactId>spring-boot-dependencies</artifactId>` bom using the spring.boot.version

**Review by caalador — APPROVED** [review]

**Pull request merged**